google/cloud/firestore_v1/document.py

```diff
@@ -107,7 +107,7 @@ def set(self, document_data, merge=False) -> Any:
         write_results = batch.commit()
         return _first_write_result(write_results)
 
-    def update(self, field_updates, option=None) -> Any:
+    def update(self, field_updates, option=None) -> Coroutine:
```

Comment:
This method returns a single `google.cloud.proto.firestore.v1.write.WriteResult` instance.
```suggestion
    def update(self, field_updates, option=None) -> write.WriteResult:
```

Reply:
Same error as mentioned in above comment